backend/api/quivr_api/modules/knowledge/service/knowledge_service.py

```diff
@@ -23,27 +24,40 @@ def __init__(self, repository: KnowledgeRepository):
         self.repository = repository
 
     async def add_knowledge(
-        self, knowledge_to_add: CreateKnowledgeProperties
+        self,
+        knowledge_to_add: CreateKnowledgeProperties,  # FIXME: (later) @Amine brain id should not be in CreateKnowledgeProperties but since storage is brain_id/file_name
```

Comment:
We should be careful here. If we return a KnowledgeDB object you should refresh the object using `await self.session.refresh(obj)` to have the inserted object fields